dnsstresss.go

```diff
@@ -47,37 +59,35 @@ func main() {
 
 	flag.Parse()
 
-	// We need exactly one parameter (the target domain)
-	if flag.NArg() != 1 {
+	// We need at least one target domain
+	if flag.NArg() < 1 {
 		flag.Usage()
 		os.Exit(1)
 	}
+	// all remaining parameters are treated as domains to be used in round-robin in the threads
+	targetDomains := flag.Args()
 
-	// The target domain should be the first (and only) parameter
-	targetDomain := flag.Args()[0]
-
-	fmt.Printf("Queried domain is %s.\n", targetDomain)
+	fmt.Printf("Queried domains are %v.\n", targetDomains)
```

Comment:
Replace `Queried domains are %v` by `Queried domain(s): %v` because using dnsstresss with only one domain may be the most probable use-case (?)